spec/filters/ruby_spec.rb

```diff
@@ -56,7 +56,7 @@
         # non-pretty json does not contain \n
         insist { subject.get("pretty").count("\n") } == 0
         # usage of JSON.parse here is to avoid parser-specific order assertions
-        insist { JSON.parse(subject.get("pretty")) } == JSON.parse("{\"message\":\"hello world\",\"mydate\":\"2014-09-23T00:00:00-0800\",\"@version\":\"1\",\"@timestamp\":\"2014-09-23T08:00:00.000Z\"}")
+        insist { JSON.parse(subject.get("pretty")) } == JSON.parse("{\"message\":\"hello world\",\"mydate\":\"2014-09-23T00:00:00-0800\",\"@version\":\"1\",\"@timestamp\":\"#{get_logstash_timestamp("2014-09-23T08:00:00.000Z")}\"}")
```

Comment:
fix the test with Logstash 8 nanosecond-precision timetamps
https://github.com/elastic/logstash/pull/12797